content/documentation/device-examples/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Devices examples
+title: Device examples
```

Comment:
are you sure? there are multiple devices

Reply:
yes, i'm sure. Like this, it will reflect the menu link